### PR TITLE
105805 component page

### PR DESCRIPTION
**Pull request by oleterno-equinor**

### CODE REVIEW

File: src/components/component/component-bread-crumb.js [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
-        {envName}
-      </Breadcrumbs.Breadcrumb>
-      <Breadcrumbs.Breadcrumb>{componentName}</Breadcrumbs.Breadcrumb>
-    </Breadcrumbs>
+    <Breadcrumb
+      links={[
+        { label: appName, to: routeWithParams(routes.app, { appName }) },
+        { label: 'Environments', to: routing.getEnvsUrl(appName) },
+        {
+          label: <EnvironmentBadge envName={envName} />,
```
Comment (nilsgstrabo) [inline src/components/component/component-bread-crumb.js:16]: I don't think we should have special formatting for environment breadcrumb.

Comment (oleterno-equinor) [reply src/components/component/component-bread-crumb.js:16]: good point, I'll change this.

File: src/components/component/env-variables.js [hunk 2]
```diff
@@ -39,20 +41,21 @@ const EnvVariables = ({ component, includeRadixVars }) => {
...
+            ( <Logo height="24px" width="24px" /> automatically added by Radix )
           </Typography>
-        ))}
-      {envVarNames.length > 0 && (
+        )}
+      </div>
+      {envVarNames.length === 0 ||
+      (!includeRadixVars && envVarNames.length - radixVarNames === 0) ? (
+        <Typography variant="body_short">
+          This {component.type === 'job' ? 'job' : 'component'} uses no
```
Comment (nilsgstrabo) [inline src/components/component/env-variables.js:55]: component type is defined in /models/component-type/index.js
Perhaps better to import the componentType const from this file and use `component.type === componentType.job` instead?

File: src/components/environments-summary/environment-card.js [hunk 3]
```diff
@@ -64,7 +64,7 @@ const EnvironmentCard = ({ appName, env }) => {
     <div className="env_card">
       <div className="env_card_header">
         <div className="header">
-          <h6>
+          <Typography variant="accordion_header" group="ui">
```
Comment (nilsgstrabo) [inline src/components/environments-summary/environment-card.js:67]: Is accordion_header a valid variant? Didn't see it in storybook, but perhaps storybook is outdated.

Comment (oleterno-equinor) [reply src/components/environments-summary/environment-card.js:67]: accordion_header is a part of the UI group in EDS. See [here](https://github.com/equinor/design-system/tree/master/libraries/core-react/src/components/Typography) that we can use the group property to render any typography style. All typography styles can bee seen h[ere in figma](https://www.figma.com/file/0TbIXrrObWj80Cf7KucKYFL0/Design-Tokens?node-id=486%3A0&viewport=-3466%2C324%2C0.505901038646698).

**Review by nilsgstrabo — APPROVED** [review]

**Pull request merged**